packages/client/package.json

```diff
@@ -12,7 +12,8 @@
   },
   "scripts": {
     "build": "tsc --project tsconfig.json",
-    "watch": "tsc --project tsconfig.json --watch",
+    "tsc": "tsc --project tsconfig.json --noEmit",
+    "tsc:watch": "tsc --project tsconfig.json --noEmit --watch",
```

Comment:
seems nice, a good improvement for the future would be to run all of them except SM in watch mode.
i'm sure @bapmrl would have nice inputs about it.